indico/modules/events/management/util.py

```diff
@@ -182,3 +183,38 @@ def _process_attachments(folders):
 
     else:
         raise TypeError('Unexpected object of type {}: {}'.format(type(root).__name__, root))
+
+
+def find_unregistered_users(person_list):
+    """Get a set of users which don't have an Indico account
+
+        :param person_list: List of persons to check
+
+    """
+    return set(filter(lambda p: p.person.user is None, person_list))
+
+
+def flash_if_unregistered(former_person_list, new_person_list, event):
+    """ Flash a message when user(s) with no Indico account is added for a contribution or as a chairperson
+
+        :param former_person_list: List of persons before new users were added
+        :param new_person_list: List of persons with new users
+        :param event: Current event
+
+    """
+    added_non_users = len(find_unregistered_users(new_person_list) - former_person_list)
+    if added_non_users > 0:
```

Comment:
Im making a set from difference of the 2 sets (new_person_list and former_person_list), to see what are the new people with no account, and then I need length of that set to display the number, so I dont know how to do it without len()